hazelcast/src/test/java/com/hazelcast/map/EvictionTest.java

```diff
@@ -132,34 +119,30 @@ public void testMaxIdle_backupEntryShouldNotBeReachableAfterMaxIdle() {
         instanceA.getMap("Test")
                 .put(keyOwnedByInstanceA, "value0", 0, SECONDS, 3, SECONDS);
 
-        assertTrueEventually(new AssertTask() {
-            @Override
-            public void run() {
-                RecordStore recordStore = getRecordStore(instanceB, keyOwnedByInstanceA);
-                assertEquals(0, recordStore.size());
-            }
+        assertTrueEventually(() -> {
+            RecordStore recordStore = getRecordStore(instanceB, keyOwnedByInstanceA);
+            assertEquals(0, recordStore.size());
         });
     }
 
     @Test
     public void testMaxIdle_backupRecordStore_mustBeExpirable() {
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
-        HazelcastInstance instance = factory.newHazelcastInstance(getConfig());
-
-        final HazelcastInstance instanceB = factory.newHazelcastInstance(getConfig());
+        Config config = getConfig()
+                .setProperty(PROP_TASK_PERIOD_SECONDS, String.valueOf(MAX_VALUE));
```

Comment:
why are we changing value of `PROP_TASK_PERIOD_SECONDS`?

Reply:
Because, I assume, this test is to verify that the record is stored and the store is marked as expirable. I wanted to make sure that multiple hiccups of over 10 seconds won't invalidate this.

Reply:
ok, makes sense.